modules/admin_manual/pages/installation/quick_guides/ubuntu_22_04.adoc

```diff
@@ -0,0 +1,328 @@
+= Install ownCloud on Ubuntu 22.04
+:toc: right
+
+== Introduction
+
+This is a short guide to installing ownCloud on a fresh installation of Ubuntu 22.04. Run the following commands in your terminal to complete the installation.
+
+This guide can not go into details and has its limits by nature. If you experience issues like with dependencies of PHP or other relevant things like the operating system, web server or database, look at the xref:installation/manual_installation/manual_installation.adoc#ubuntu-22-04-lts-server[Detailed Installation Guide] for more information.
+
+== Prerequisites and Notes
+
+* A fresh installation of https://www.ubuntu.com/download/server[Ubuntu 22.04] with SSH enabled.
+* This guide assumes that you are working as the root user.
+* Your ownCloud directory will be located in `/var/www/owncloud/`.
+* php 7.4 from the ondrej/php PPA is going to be installed on Ubuntu 22.04.
+* Use the correct xref:installation/manual_installation/manual_installation_apache.adoc#multi-processing-module-mpm[Multi-Processing Module (MPM)].
+
+== Preparation
+
+=== Set Your Domain Name
+
+[source,bash]
+----
+my_domain="Your.Domain.tld"
+echo $my_domain
+
+hostnamectl set-hostname $my_domain
+hostname -f
+----
+
+=== Generate Strong Passwords
+
+----
+sec_admin_pwd=$(openssl rand -base64 18)
+echo $sec_admin_pwd > /etc/.sec_admin_pwd.txt
+
+sec_db_pwd=$(openssl rand -base64 18)
+echo $sec_db_pwd > /etc/.sec_db_pwd.txt
+----
+
+=== Update Your System
+
+First, ensure that all the installed packages are entirely up to date and that PHP is available in the APT repository.
+To do so, follow the instructions below:
+
+[source,bash]
+----
+apt update && \
+  apt upgrade -y
+----
+
+=== Create the occ Helper Script
+
+Create a helper script to simplify running xref:configuration/server/occ_command.adoc[occ commands]:
+
+[source,bash]
+----
+FILE="/usr/local/bin/occ"
+cat <<EOM >$FILE
+#! /bin/bash
+cd /var/www/owncloud
+sudo -E -u www-data /usr/bin/php /var/www/owncloud/occ "\$@"
+EOM
+----
+
+Make the helper script executable:
+
+[source,bash]
+----
+chmod +x $FILE
+----
+
+=== Install the Required Packages
+
+[source,bash]
+----
+sudo add-apt-repository ppa:ondrej/php -y
+sudo apt update && sudo apt upgrade
+
+apt install -y \
+  apache2 \
+  libapache2-mod-php7.4 \
+  mariadb-server \
+  openssl redis-server wget \
+  php7.4 php7.4-imagick php7.4-common \ 
+  php7.4-curl php7.4-gd php7.4-imap \
+  php7.4-intl php7.4-json php7.4-mbstring \
+  php7.4-mysql php7.4-ssh2 php7.4-xml \
+  php7.4-zip php7.4-apcu php7.4-redis \
+  php7.4-ldap php7.4-opcache php7.4-phpseclib\
```

Comment:
On Ubuntu 22.04 with Ondrej repos I cannot find php7.4-phpseclib

Should that be `php-phpseclib` instead? That exists, but is probably meant for php8, not sure if that also works with 7.4 ...

Reply:
@jnweiger You are right, `php7.4-phpseclib` does not exist on Ondrej repo, I may have done an error while documenting. Not sure if we want to document at this point `php-phpseclib` as required package

Reply:
Note, seems that php-seclib is part of the ondrej package, see: https://github.com/oerdnj/deb.sury.org/issues/445
Needs clarification and raises the question now is how is it named so it can be included properly...?

Reply:
ok, I think I got the solution 😄 and I will update the PR accordingly

From: https://phpseclib.com/docs/why
```
Why phpseclib?
phpseclib provides pure-PHP implementations of SSH2, SFTP, RSA, DSA, Elliptic Curves, AES, ChaCha20, X.509, CSR, CRL, SPKAC, etc.

Portability
The only requirement that phpseclib 3.0 has is that you must be using PHP 5.6+.

Extensions like bcmath, gmp, libsodium and openssl, if they're available, for speed, but they're not required.
```

And when installing it with `sudo apt install php-phpseclib` you will find the follonwing files:
```
dpkg -L php-phpseclib

/.
/usr
/usr/share
/usr/share/doc
/usr/share/doc/php-phpseclib
/usr/share/doc/php-phpseclib/changelog.Debian.gz
/usr/share/doc/php-phpseclib/copyright
/usr/share/doc/php-phpseclib/examples
/usr/share/doc/php-phpseclib/examples/openssl.cnf
/usr/share/php
/usr/share/php/phpseclib
/usr/share/php/phpseclib/Crypt
/usr/share/php/phpseclib/Crypt/AES.php
/usr/share/php/phpseclib/Crypt/Base.php
/usr/share/php/phpseclib/Crypt/Blowfish.php
/usr/share/php/phpseclib/Crypt/DES.php
/usr/share/php/phpseclib/Crypt/Hash.php
/usr/share/php/phpseclib/Crypt/RC2.php
/usr/share/php/phpseclib/Crypt/RC4.php
/usr/share/php/phpseclib/Crypt/RSA.php
/usr/share/php/phpseclib/Crypt/Random.php
...
```

Reply:
update applied, all good